pep/app/pods/components/modal-dialogs/user/about/template.hbs

```diff
@@ -0,0 +1,24 @@
+<ModalDialog @onClose={{@onClose}} @size="md" as |Modal|>
+    <Modal.header @title={{t "about.title"}} />
+    <Modal.body>
+        <div class="mb-2">
+            <div>{{t "about.client"}}</div>
+            <div class="text-muted small">{{@options.clientBuildVersion}}</div>
+        </div>
+        <div class="mb-2">
+            <div>{{t "about.server"}}</div>
+            <div class="text-muted small">{{@options.serverInformation.text_server_version}}</div>
+            <div class="text-muted small">{{@options.serverInformation.dataSource}}</div>
+        </div>
+    </Modal.body>
+    <Modal.footer>
+        <div class="d-flex justify-content-between w-100 align-items-center">
```

Comment:
you should be able to remove the need for this inner div container, and just do this right on the footer component:

```hbs
<Modal.footer class="justify-content-between align-items-center">
    ...
</Modal.footer>
```
and should no longer need `d-flex` or `w-100` in this case too.